provider/provider.go

```diff
@@ -942,15 +946,24 @@ func parseRelaySuccesfulOutput(bodyBytes []byte, requestStatusCode int) (*RelayO
 		return &output, err
 	}
 
-	// Check if there's explicitly a result field, if there's on mark it as success, otherwise check what's the potential status.
-	// for REST chain that doesn't return result in any of the call will be defaulted to 202 in extractStatusFromResponse
+	isValidJson := json.Valid([]byte(output.Response))
```

Comment:
My only question is an open one for @adshmh : does this impact the Portal logic for handling Pocket Mainnet/Archival and if so, how much of an impact is this?

Reply:
+1 to any insight you have.

I tried to be as backwards compatible as possible with the if/else below, but it only uses logic and not any sort of real testing.

Reply:
I believe there are cases where Pocket returns a non-json response, but those may have the text `result` in them (we need to verify this)

Reply:
@adshmh so in terms of next step with this PR, which is it?

1. Merge it in -> Update portal-middleware -> deploy -> test in prod
OR
2. Do some sort of investigation for different request/response types?
OR
3. 👍 that this is backwards compatible enough and go with option (1) above

I'm opting for option (3) but lmk if you have a better idea for (2).

Please approve/request changes to the PR as needed.

Reply:
I agree that option [3] makes the most sense here. 
To verify backward compatibility of the change we need to a) review the API docs and b) run a few tests against a Morse node.

Other than potential compatibility issues, the code changes look OK.

Reply:
> review the API docs

This is correct: https://docs.pokt.network/reference/api-definition

Here is the source of truth: https://github.com/pokt-network/pocket-core/blob/staging/doc/specs/rpc-spec.yaml

> run a few tests against a Morse node.

Sounds good. 

1. Where are the instructions to do this?
2. Is this something you can do?
3. Happy to dive into it if (1) does not exist or you don't have the capacity given (2).

Reply:
1. I don't think this has been done/documented before.
2. Yes: I can build a doc and run the tests.